packages/react/src/components/AnimatedHeader/components/AnimatedHeader/AnimatedHeader.tsx

```diff
@@ -231,7 +233,6 @@ const AnimatedHeader: React.FC<AnimatedHeaderProps> = ({
                   id={`${blockClass}__header-dropdown`}
                   className={`${blockClass}__header-dropdown`}
                   size="md"
-                  autoAlign
                   titleText="Label"
                   initialSelectedItem={allTiles[0]}
                   label={allTiles[0].name || 'Select an option below'}
```

Comment:
What do you think about exporting this label as prop as well? It will solve the internationalisation problems for it and give move flexibility. We have a requirement to have a custom label non-dependant on first tile name.

Reply:
Can you show an example where the label wouldn't represent the Tile Group that is selected? 

I just want to understand logically how this would work. In the current setup, the Dropdown label represents the tiles/items selected to the right. 

Since the labels are coming from the Tile data there should be no issues with internationalization since the data is directly coming from the end user. The 'Select an option below' is the backup option which is English only currently.